retrieverdash/README.rst

```diff
@@ -1,46 +1,36 @@
-django-project-skeleton
-=======================
+Retriever Status Dashboard
+==========================
+
+**Retriever Status Dashboard** provides details and status info.... 
```

Comment:
**Retriever Status Dashboard** provides details and status info of datasets available through **retriever**.